docs/academic/README.md

```diff
@@ -0,0 +1 @@
+Placeholder for all academic documentation and related material!
```

Comment:
?

Reply:
Deleted this. Just a placeholder as I used the git interface to upload it